resources/x509_certificate.rb

```diff
@@ -0,0 +1,134 @@
+provides :openssl_x509 # legacy_name
+provides :openssl_x509_cert
+provides :openssl_x509_certificate
+
+include OpenSSLCookbook::Helpers
+
+property :path,             String, name_property: true
+property :owner,            String, default: node['platform'] == 'windows' ? 'Administrator' : 'root'
```

Comment:
You can shrink this a bit with platform?('windows') ? 'Administrator' : 'root'

Reply:
Will do it tomorrow. ;-)

Reply:
I can just do it now. Easy enough.